src/types/ButtonType.ts

```diff
@@ -0,0 +1 @@
+export type ButtonType = 'prev' | 'next';
```

Comment:
It seems to me keeping common types/interfaces is ok in one file "models" to decrease number of files and simpify access. The same with enums/constants